src/middlewared/middlewared/plugins/apps/ix_volumes.py

```diff
@@ -14,9 +14,9 @@ class Config:
         namespace = 'app.ix_volume'
         event_send = False
         cli_namespace = 'app.ix_volume'
-        entry = AppIXVolumeEntry
+        entry = AppsIxVolumeEntry
 
-    @filterable_api_method(item=AppIXVolumeEntry, roles=['APPS_READ'])
+    @filterable_api_method(item=AppsIxVolumeEntry, roles=['APPS_READ'])
```

Comment:
```suggestion
    @filterable_api_method(item=AppIxVolumeEntry, roles=['APPS_READ'])
```
same for the other models